### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by loganthomas**
# PR Description

Please describe the changes proposed in the pull request:

- Replaces references to https://github.com/ericmjl/pyjanitor/ with https://pyjanitor-devs.github.io/pyjanitor/
- Updates `AUTHORS.rst` to reflect new pyjanitor-dev leads
- Updates `pivot_wider()` so that only top line in docstring is rendered. Currently, it looks like the below image:

Screen Shot 2021-08-11 at 22 11 43

- Late addition: strips `as_categorical()` references in light of `encode_categorical()`

# PR Checklist

Please ensure that you have done the following:

1. [X] PR in from a fork off your branch. Do not PR from `<your_username>`:`dev`, but rather from `<your_username>`:`<feature-branch_name>`.

2. [X] If you're not on the contributors list, add yourself to `AUTHORS.rst`.

3. [X] Add a line to `CHANGELOG.md` under the latest version header (i.e. the one that is "on deck") describing the contribution.
    - Do use some discretion here; if there are multiple PRs that are related, keep them in a single line.

# Automatic checks

There will be automatic checks run on the PR. These include:

- Building a preview of the docs on Netlify
- Automatically linting the code
- Making sure the code is documented
- Making sure that all tests are passed
- Making sure that code coverage doesn't go down.

# Relevant Reviewers

Please tag maintainers to review.

- @ericmjl
- @samukweku 
- @nvamsikrishna05

### CODE REVIEW

**Comment by loganthomas** [comment]
@ericmjl and @samukweku
One thing I noticed when building the docs, `janitor.as_categorical` is no longer a function. Did this get replaced with`janitor.encode_catecorical`?

Error message:
```
WARNING: [autosummary] failed to import 'janitor.as_categorical': no module named janitor.as_categorical
```

If this is true, I'd like to clean this up within this PR. Thanks!

**Review by samukweku — APPROVED** [review]
@loganthomas yes, as_categorical is superseded by encode_categorical.

**Comment by samukweku** [comment]
@loganthomas it seems the docs don't align with the most recent docs in the repo.

**Comment by loganthomas** [comment]
> @loganthomas yes, as_categorical is superseded by encode_categorical.

Thanks @samukweku! Updated and stripped ✂️

**Comment by loganthomas** [comment]
> @loganthomas it seems the docs don't align with the most recent docs in the repo.

@samukweku Are you building the docs locally? Or, do you mean from https://pyjanitor-devs.github.io/pyjanitor/?

**Comment by samukweku** [comment]
@loganthomas, locally, everything renders fine, but when I view it from https://pyjanitor-devs.github.io/pyjanitor/, the content is not up to date with `dev`.

By the way, I am learning from your updates on this ticket (learning a lot about `devops` through your work!)

**Comment by loganthomas** [comment]
@samukweku can you provide an example of the docs that aren't aligned? This PR hasn't been merged to `dev` so the changes won't be reflected in production yet.

**Comment by samukweku** [comment]
@loganthomas no worries; I'm sure it will be resolved after the merge. thanks!

File: AUTHORS.rst [hunk 1]
```diff
@@ -29,6 +29,9 @@ Leads
     - Started pyspark sub-module.
 - `@anzelpwj <https://github.com/anzelpwj>`_ | `contributions <https://github.com/pyjanitor-devs/pyjanitor/pulls?utf8=%E2%9C%93&q=is%3Aclosed+mentions%3Aanzelpwj>`_
 - `@samukweku <https://github.com/samukweku>`_ | `contributions <https://github.com/pyjanitor-devs/pyjanitor/pulls?utf8=%E2%9C%93&q=is%3Aclosed+mentions%3Asamukweku>`_
+- `@loganthomas <https://github.com/loganthomas>`_ | `contributions <https://github.com/pyjanitor-devs/pyjanitor/issues?q=is%3Aclosed+mentions%3Aloganthomas>`_
+    - Helped others with ``git`` issues at SciPy 2019.
+- `@nvamsikrishna05 <https://github.com/nvamsikrishna05>`_ |  `contributions <https://github.com/pyjanitor-devs/pyjanitor/issues?q=is%3Aclosed+mentions%3Anvamsikrishna05>`_
```
Comment (ericmjl) [inline AUTHORS.rst:34]: 🎉 thank you for handling this, @loganthomas!

**Review by ericmjl — APPROVED** [review]

**Comment by ericmjl** [comment]
Not worried about Codecov for the time being, looks minor to me. Let's merge!

**Comment by ericmjl** [comment]
Btw, I think netlify page previews are broken, will see if I can get to it tomorrow.

**Comment by loganthomas** [comment]
@samukweku @ericmjl thank you both for the review!

**Pull request merged**